testpilot/frontend/static-src/app/templates/landing-page.js

```diff
@@ -9,7 +9,12 @@ export default `
             <span data-l10n-id="siteName">Firefox Test Pilot</span>
           </a>
         </h1>
-        <a data-l10n-id="landingFxaAlternateButton" href="/accounts/login/?next=/" class="button outline">Sign in</a>
+        {{^isFirefox}}
+          <span></span>
```

Comment:
@pdehaan no, needs an empty empty span to maintain the correct flexing, although i suppose it could be an empty tag.